stups-spring-oauth2-server/README.md

```diff
@@ -44,3 +44,43 @@ public class OAuthConfiguration extends ResourceServerConfigurerAdapter {
 
 }
 ```
+
+You can also configure a HttpUserRolesProvider which will allow more advanced spring security access policies.
```

Comment:
A little bit more information would be awesome. What does it solve. Why do I want to use roles and not oauth scopes? What's the usecase? Why is it used at Zalando?

The whole approach is pretty awesome and helps to solve the problem that "persons" don't have oauth scopes (only services have for now).  We should make sure anyone reading the docs understand the rationale "Why" that has been implemented.